lang/LangSource/PyrObject.cpp

```diff
@@ -1374,9 +1373,6 @@ void buildBigMethodMatrix()
 			100. * (double)numFilled/(rowTableSize/sizeof(PyrMethod*)));
 	}
 #endif
-	//post("\t%d method selectors, %d classes\n", numSelectors, numClasses);
-	//post("\tmethod table size %d bytes, ", rowTableSize);
-	//post("big table size %d\n", numSelectors * numClasses * sizeof(PyrMethod*));
 	//postfl("%p %p %p\n", classes, bigTable, sels);
```

Comment:
method table size tells you how many methods there are and what size the lookup table for that is. These do affect performance and memory usage so somebody might want to know them. Most of the time we don't, so we can hide them behind a flag.